libnmstate/nm/ipv6.py

```diff
@@ -158,3 +165,110 @@ def get_ip_profile(active_connection):
     if remote_conn:
         return remote_conn.get_setting_ip6_config()
     return None
+
+
+def get_route_info():
+    ret = []
+    client = nmclient.client()
+    for active_connection in client.get_active_connections():
+        if active_connection is None:
+            continue
+        ip_cfg = active_connection.get_ip6_config()
+        if not ip_cfg:
+            continue
+
+        devs = active_connection.get_devices()
```

Comment:
This code seems to be the same as in the IPv4 module. Would be great to move it to a shared function.

Reply:
Edy seems don't allow me to create a `route.py` to hold this function as he want to align with NM data structure in `libnmstate/nm` folder.

Reply:
You can create such a module, but I think you should expose it through the ipX modules as well, just to keep a sync with nm structure.

Maybe at the ipX modules detect what nm object is needed, then pass it with the data to a route module. So, common stuff in route and specific (6/4) in ipX.

Reply:
If the module can be generic for IP a name like `ip.py` makes more sense. Cannot remember that people use `ipX` to say that it can be any version of IP. It sounds more like IPX.

Reply:
I have moved the codes to `route.py`, the `ipv4.py` and `ipv6.py` only provide the function reference to `route.py` to use the unified codes.